modules/gdscript/config.py

```diff
@@ -15,3 +15,9 @@ def get_doc_classes():
 
 def get_doc_path():
     return "doc_classes"
+
+
+def get_module_dependencies():
+    return [
+        "jsonrpc",
+    ]
```

Comment:
Seems to depend on websocket too.

Reply:
Both text servers can be built without `freetype` (will support bitmap fonts only) and `msdfgen` (no MSDF rendering, pre-rendered MSDF will work), so it should not be a dependency. But it might be useful to auto disabling `freetype` and `msdfgen` if none of text servers is enabled (also valid config, if you do not want any text for some reason).